docusaurus/docs/cms/plugins-development/widgets-api.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Widgets API
+description: Learn how to use the Widgets API to create and manage widgets in the Strapi admin panel.
+displayed_sidebar: cmsSidebar
+tags:
+  - widgets
+  - admin panel
+  - plugins development
+---
+
+# Widgets API
+
+The Widgets API allows you to create and manage widgets in the Strapi admin panel. Widgets are customizable components that can be added to various parts of the admin interface to display information or provide additional functionality.
+
+## Creating a Widget
+
+To create a widget, you need to use the `Widgets` class from the `@strapi/admin` package. Here's an example of how to create a widget:
+
```

Comment:
This is wrong, what we should explain instead is:
- they should register the widgets during Strapi's register phase
- instead of importing anything, they should access the widgets register api via the app object provided as a param of the Strapi's register lifecycle. [Exactly like custom fields](https://docs.strapi.io/cms/plugins-development/custom-fields#registering-a-custom-field-in-the-admin-panel)